src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -494,6 +526,12 @@ void BLEManagerImpl::HandleBLEAdvertisementTimeout(System::Layer * layer, void *
     ChipLogProgress(DeviceLayer, "CHIPoBLE advertising disabled because of timeout expired");
 }
 
+void BLEManagerImpl::HandleBLEAdvertisementIntervalChange(System::Layer * layer, void * param, System::Error error)
+{
+    BLEMgr().SetAdvertisingNeedsRefresh();
+    ChipLogProgress(DeviceLayer, "CHIPoBLE advertising interval changed");
```

Comment:
Perhaps we should print such a log when the change actually happens, not when we schedule the change. Thoughts?

Reply:
Actually I changed log to inform about changing advertising mode, so now it should be more relevant.